ray-operator/main.go

```diff
@@ -64,6 +64,8 @@ func main() {
 	var logFileEncoder string
 	var logStdoutEncoder string
 	var configFile string
+	var EnableBatchScheduler bool
```

Comment:
Should the `EnableBatchScheduler` and `ForcedClusterUpgrade` variables be removed from the `raycluster_controller.go` file?